src/HolographicCamera.Unity/Assets/HolographicCamera/Scripts/KeepDeviceActive.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+#if UNITY_WSA && WINDOWS_UWP
+    using System;
+    using Windows.System.Display;
+#endif
+
+public class KeepDeviceActive : MonoBehaviour
```

Comment:
Add a doccomment explaining what this class does.

It's actually a super useful class in general given the API that it invokes.

Reply:
sounds good